config/repo-values.yaml

```diff
@@ -92,3 +92,5 @@ sync:
       url: https://helm.mattermost.com
     - name: emberstack
       url: https://emberstack.github.io/helm-charts
+    - name: athens
```

Comment:
I take this back! Since I can't change the chart URL to `https://athens.blob.core.windows.net/charts` at the moment, let's make it `gomods`. Sorry for the confusion @rimusz

Reply:
done